Playmate/View Controllers/Profile Tab/Friends/IncomingRequestsViewController.m

```diff
@@ -37,7 +38,9 @@ - (void)viewWillAppear:(BOOL)animated {
     
     // set up refresh control
     self.refreshControl = [[UIRefreshControl alloc] init];
-    [self.refreshControl addTarget:self action:@selector(fetchData) forControlEvents:UIControlEventValueChanged];
+    [self.refreshControl addTarget:self
+                         action:@selector(fetchData)
```

Comment:
usually, when you break a function to several lines, you want to keep the `:` aligned in the same column. 

I am happy to jump into a VC, to explain myself better.

Reply:
Ahh that makes a lot of sense. I understand what you mean by aligning the colons, but was there anything else you wanted to explain more in depth?